pkg/apis/postgres-operator.crunchydata.com/v1beta1/postgrescluster_types.go

```diff
@@ -186,6 +186,7 @@ type ExtensionsSpec struct {
 	PGStatMonitor    bool `json:"pgStatMonitor,omitempty"`
 	PGAudit          bool `json:"pgAudit,omitempty"`
 	PGStatStatements bool `json:"pgStatStatements,omitempty"`
+	PGVector         bool `json:"pgvector,omitempty"`
```

Comment:
Should we use camelCase for pgvector as we have for other extensions, like `pgVector`?